### PR TITLE
JWT Scopes

### PR DESCRIPTION
**Pull request by sedkis**
Add section about how to manage scopes and associate them with policies when using a JWT protected API and JWT tokens

### CODE REVIEW

**Comment by sedkis** [comment]
Yikes, this was forked off another feature branch.  Which is why it includes that CSS update section.  Please ignore that while I fix it.

File: tyk-docs/content/tyk-developer-portal/customise/customising-using-dashboard.md [hunk 1]
```diff
@@ -26,6 +26,41 @@ In the CSS Editor, add the classes that you would like to override in the home p
...
+Alternatively, as always, you can perform the above actions with an API call instead of through the Dashboard UI.
+
+First, we'll need to get the block ID of the CSS component in order to update it.  This is stored in Mongo by the Dashboard.
+To get the block ID, we have to make a REST call to the Dashboard API.  
+
+To do so, run this `curl` command:
+
+```{.copyWrapper}
+curl www.tyk-test.com:3000/api/portal/css \
+-H "Authorization:<DASHBOARD API KEY>"
```
Comment (letzya) [inline tyk-docs/content/tyk-developer-portal/customise/customising-using-dashboard.md]: @marksou what is our recommended notation for variables? 
 -H "Authorization:<DASHBOARD API KEY>"  
or 
 -H "Authorization:{DASHBOARD API KEY}" 
or something else?

Comment (sedkis) [reply tyk-docs/content/tyk-developer-portal/customise/customising-using-dashboard.md]: This is a good question.. we definitely need to define a standard.

Comment (marksou) [reply tyk-docs/content/tyk-developer-portal/customise/customising-using-dashboard.md]: I've been using (for example):

-H "Authorization:{DASHBOARD_API_KEY}"

So using uppercase and underscores with curly brackets.

**Review by letzya — CHANGES_REQUESTED** [review]
Remove jwt completely (will be part of 2.9 release).

**Comment by sedkis** [comment]
@letzya perhaps we can leave this open until we merge the CSS change in its own PR:
https://github.com/TykTechnologies/tyk-docs/pull/842
once 842 gets merged we can leave this PR open until 2.9, then we can merge it
edit: @letzya what do u think ?

**Comment by letzya** [comment]
@sedkis a bit confused and don’t want to compare. Can you maybe just  remove the css changes from here and add them to the other pr?

**Comment by sedkis** [comment]
Closing so we don't accidentally merge.  but we can use this to build onto

**Comment by letzya** [comment]
I’ll write something next week

**Pull request closed without merging**